django/contrib/sitemaps/__init__.py

```diff
@@ -122,6 +124,13 @@ def location(self, item):
 
     def get_protocol(self, protocol=None):
         # Determine protocol
+        if not any((self.protocol, protocol)):
+            warnings.warn(
+                "http as the default sitemap protocol is deprecated. "
+                "https will be the new default protocol. "
+                "Set an explicit 'protocol' to silence this warning.",
+                category=RemovedInDjango41Warning
+            )
         return self.protocol or protocol or 'http'
```

Comment:
Add a comment above this line to explain what needs to be changed after the deprecation period has expired:
```suggestion
        # RemovedInDjango50Warning: Change 'http' to 'https' after deprecation.
        return self.protocol or protocol or 'http'
```